### PR TITLE
Nathaniel

### PR DESCRIPTION
**Pull request by bugeybot**
Full tested working set of autos, pnuematics, drive train, and drive controls.

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -52,76 +105,94 @@ public void robotInit() {
...
-   * the robot is disabled.
+   * This function is called when the disabled button is hit.
+   * You can use it to reset subsystems before shutting down.
    */
-  @Override
-  public void disabledInit() {
-  }
+	public void disabledInit(){
+		System.out.println("At " + ((System.currentTimeMillis() - initialBootTime) * 1000) + ", robot19.robot says \n"
+				+ "\" WE ARE DISABLED WHAT THE HECK?\"");
```
Comment (122004) [inline src/main/java/frc/robot/Robot.java:115]: Not very professional. Please use more exclamation points.

**Review by 122004 — CHANGES_REQUESTED** [review]
Needs more exclamation points when the system is disabled. It should convey the general panic we feel as all things start to go wrong. Other than that, perfect.

**Review by 122004 — APPROVED** [review]
Disappointed that it is not named AirBoi.

**Review by Linkd-Aether — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/Drivetrain.java [hunk 2]
```diff
@@ -1,56 +1,55 @@
...
 
 public class Drivetrain extends Subsystem {
-
+    
     private final WPI_TalonSRX rightMotor1 = RobotMap.rightMotor1;
     private final WPI_VictorSPX rightMotor2 = RobotMap.rightMotor2;
     private final WPI_TalonSRX leftMotor1 = RobotMap.leftMotor1;
-    private final WPI_VictorSPX leftMotor2 = RobotMap.leftMotor2;
-
+	private final WPI_VictorSPX leftMotor2 = RobotMap.leftMotor2;
```
Comment (122004) [inline src/main/java/frc/robot/subsystems/Drivetrain.java:17]: sPAcInG is off.

**Review by 122004 — COMMENTED** [review]
You just copied a bunch of stuff on robot 18 that was already there...

**Pull request merged**